app/assets/stylesheets/michigan-benefits/organisms/_main-footer.scss

```diff
@@ -0,0 +1,14 @@
+.main-footer {
+  font-weight: 300;
+}
+
+.main-footer {
+  .illustration--cfa, .illustration--civilla {
+    float: left;
+    margin-top: 46px;
```

Comment:
would be ideal to not use such a specific margin height if we can (prefer rem or referencing a variable)